### PR TITLE
Fix disabled tests.

### PR DESCRIPTION
**Pull request by mikepizzo**
Teach Taupo to read primitives and collections within collections.

### Issues

*This pull request fixes issue #1427*

### Description

Fix tests disabled for streaming check-in.

### Checklist (Uncheck if it is not completed)

- [x] *Test cases added*
- [x] *Build and test with one-click build and test script passed*

### CODE REVIEW

File: test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs [hunk 1]
```diff
@@ -402,6 +402,21 @@ private QueryStructuralValue BuildFromEntityInstance(EntityInstance instance, Qu
             return entity;
         }
 
+        /// <summary>
+        /// Builds an instance of QueryPrimitiveValue from an primitiveValue instance.
```
Comment (xuzhg) [inline test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs]: >QueryPrimitiveValue [](start = 34, length = 19)

QueryScalarValue

File: test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs [hunk 2]
```diff
@@ -402,6 +402,21 @@ private QueryStructuralValue BuildFromEntityInstance(EntityInstance instance, Qu
             return entity;
         }
 
+        /// <summary>
+        /// Builds an instance of QueryPrimitiveValue from an primitiveValue instance.
+        /// </summary>
+        /// <param name="value">The primitiveValue instance</param>
+        /// <param name="type">The QueryEntityType of the entity</param>
+        /// <returns>The converted structural value of entity instance.</returns>
```
Comment (xuzhg) [inline test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs]: >The converted structural value of entity instance. [](start = 21, length = 50)

rename it？

File: test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs (hunk 1 above)
Comment (mikepizzo) [reply test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs]: fixed

---
In reply to: [289951744](https://github.com/OData/odata.net/pull/1474#discussion_r289951744) [](ancestors = 289951744)

File: test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs (hunk 2 above)
Comment (mikepizzo) [reply test/FunctionalTests/Taupo/Source/Taupo.Astoria/Client/ClientResponseQueryValueBuilder.cs]: fixed

---
In reply to: [289951906](https://github.com/OData/odata.net/pull/1474#discussion_r289951906) [](ancestors = 289951906)

**Pull request merged**